change/office-ui-fabric-react-2019-07-31-08-43-04-ticks.json

```
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
+  "comment": "Fixes issues on the ticks",
```

I have a better suggestion that reflects the `minor` change you are introducing:
```suggestion
  "comment": "Slider: adds new feature of optionally rendering marks for every step of Slider control.”,
```